lib/cdo/global.rb

```diff
@@ -0,0 +1,46 @@
+module Cdo
+  # Lazily loads global configurations for regional pages
+  module Global
```

Comment:
Could we be more consistent and use one name for this feature across the codebase?
```suggestion
  module GlobalEdition
```